pkgs/by-name/ze/zed-editor/package.nix

```diff
@@ -93,14 +95,23 @@ rustPlatform.buildRustPackage rec {
     repo = "zed";
     rev = "refs/tags/v${version}";
     hash = "sha256-xtSdlzj1AxhJN4aXLJ+Oy51LX4QduLwcuCfK42kthvE=";
-    fetchSubmodules = true;
```

Comment:
This seems to have become obsolete for a while now, so I disabled fetching submodules, but correct me if I'm wrong.